src/extras/PMREMGenerator.js

```diff
@@ -261,24 +273,36 @@ class PMREMGenerator {
 		const toneMapping = renderer.toneMapping;
 		renderer.getClearColor( _clearColor );
 		const clearAlpha = renderer.getClearAlpha();
+		const originalBackground = scene.background;
 
 		renderer.toneMapping = NoToneMapping;
 		renderer.outputEncoding = LinearEncoding;
 
-		let background = scene.background;
-		if ( background && background.isColor ) {
+		const background = scene.background;
+		if ( background ) {
+
+			if ( background.isColor ) {
+
+				_backgroundColor.copy( background ).convertSRGBToLinear();
```

Comment:
One note about this...

It's pretty clear to me now that `THREE.Color` should always be linear.

We have to find all the places where it's not.

Reply:
Can you elaborate? My impression was that scene.background and renderer.clearColor were the only places that SRGB colors were expected (https://github.com/mrdoob/three.js/issues/20819#issuecomment-739066689). Are you suggesting those should be changed to Linear, as well? Are there other places SRGB is required?

Reply:
I don't know about `renderer.clearColor`, but `scene.background` should definitely be linear.
I think color spaces / encodings is a hard enough concept. Having exceptions make things ever harder.

Reply:
I'm all for updating the background colors to expect linear colors but I do think it would be very confusing if renderer.clearColor and scene.background yield different results with the same color.

Reply:
Actually, `renderer.clearColor` should be linear too.

Reply:
> but I do think it would be very confusing if renderer.clearColor and scene.background yield different results with the same color.

Indeed.

Reply:
Linear color sounds great and all, but isn't a common use case of background/clear color to match the CSS of the page around it? If tone-mapping is applied, that's going to get pretty tricky.

Reply:
Wouldn't it be better to make the background transparent?

Reply:
Yeah, that's a simple solution. Some perf cost, but probably not enough to worry about.